src/tests/class_tests/openms/source/MRMAssay_test.cpp

```diff
@@ -94,88 +107,31 @@ START_SECTION(~MRMAssay())
 {
   delete ptr;
 }
-END_SECTION
 
-START_SECTION(bool MRMAssay::isUIS_(const double fragment_ion, std::vector<double> ions, const double mz_threshold); )
-{
-  MRMAssay_test mrma;
-
-  std::vector<double> ions;
-  ions.push_back(100.00);
-  ions.push_back(100.01);
-  ions.push_back(100.12);
-  ions.push_back(101.11);
-
-  TEST_EQUAL(mrma.isUIS_test(100.065, ions, 0.05), true);
-  TEST_EQUAL(mrma.isUIS_test(100.06, ions, 0.06), false);
-}
 END_SECTION
 
-START_SECTION(void MRMAssay::isSiteUIS_(const AASequence sequence, ReactionMonitoringTransition & tr))
+START_SECTION(std::vector<std::string> MRMAssay::isUIS_(const double fragment_ion, std::vector<std::pair<double, std::string> > ions, const double mz_threshold); )
 {
   MRMAssay_test mrma;
-  MRMIonSeries mrmis;
-  AASequence sequence = AASequence::fromString("PEPT(Phospho)ID(Oxidation)EK");
-  ReactionMonitoringTransition tr1, tr2, tr3, tr4, btr1, btr2, btr3, btr4;
-
-  mrmis.annotateTransitionCV(tr1, "y5^3");
-  mrma.isSiteUIS_test(sequence, tr1);
-
-  TEST_EQUAL(tr1.getMetaValue("site_identifying_transition"), "4")
-  TEST_EQUAL(tr1.getMetaValue("site_identifying_class"), "revdiag")
-
-  mrmis.annotateTransitionCV(tr2, "y4^3");
-  mrma.isSiteUIS_test(sequence, tr2);
 
-  TEST_EQUAL(tr2.getMetaValue("site_identifying_transition"), "4")
-  TEST_EQUAL(tr2.getMetaValue("site_identifying_class"), "revnext")
+  std::vector<std::pair<double, std::string> > ions;
+  ions.push_back(std::make_pair(100.00, "PEPTIDEK"));
+  ions.push_back(std::make_pair(100.01, "PEPTIDEK"));
+  ions.push_back(std::make_pair(100.10, "PEPT(UniMod:21)IDEK"));
+  ions.push_back(std::make_pair(100.12, "PEPTIDEK"));
+  ions.push_back(std::make_pair(100.11, "PEPTIDEK"));
 
-  mrmis.annotateTransitionCV(tr3, "b4^3");
-  mrma.isSiteUIS_test(sequence, tr3);
+  std::vector<std::string> isoforms1 = mrma.isUIS_test(100.06, ions, 0.03);
```

Comment:
it seems that now `isUIS_` does not really do what the docu and the fxn name advertises, it now also returns all isoforms that could potentially generate the fragment ion. correct? maybe rename the fxn and update the documentation as well?